data/json/items/comestibles/mutagen.json

```diff
@@ -21,182 +21,83 @@
     },
     {
         "id" : "mutagen_beast",
+        "copy-from" : "mutagen",
         "type" : "COMESTIBLE",
-        "comestible_type" : "DRINK",
         "name" : "beast mutagen",
         "description" : "A rare substance of uncertain origins.  Causes you to mutate.",
```

Comment:
Cut them out, then make an issue saying that mutagens need better descriptions.

Don't bundle describing mutagens with a cleanup PR.